cmd/utils/flags.go

```diff
@@ -1564,6 +1570,9 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *eth.Config) {
 	// Ancient tx indices pruning is not available for les server now
 	// since light client relies on the server for transaction status query.
 	CheckExclusive(ctx, LegacyLightServFlag, LightServeFlag, TxLookupLimitFlag)
+
+	CheckExclusive(ctx, AncientFlag, AncientRPCFlag)
```

Comment:
Flag exclusivity is only checked here, but ancient flags are available on other commands too.

Reply:
I think we can resolve this. All commands that refer to ancient flag call SetEthConfig, which in term triggers this flag check ( inspect/copy/import) in fact all geth commands pass through the checks in SetEth.

Reply:
additionally in any of the sub projects don't use the AncientFlag

Reply:
Got it, SGTM.